### PR TITLE
Remove comment when completed, add cli command to complete analysis

### PR DESCRIPTION
**Pull request by Mropat**
This PR adds/fixes ...

- Adds command line option 
`trailblazer set-completed [ANALYSIS ID]` - Set analysis to completed by ID from command line

- Removes elapsed time from comment

### How to prepare for test
- [x] ssh to hasta (depending on type of change)
- [x] activate stage: `us`
- [x] request trailblazer-stage on hasta: `paxa`
- [x] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-trailblazer-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [x] login to hasta and us
- [x] do trailblazer set-completed <analysis_id>

### Expected test outcome
- [x] check that analysis status set to completed
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by HS
- [x] tests executed by MR
- [x] "Merge and deploy" approved by MR
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [x] **MINOR** - when you add functionality in a backwards compatible manner

### CODE REVIEW

**Comment by Mropat** [comment]
STAGE Trailblazer view
image

Run command set-completed
image

STAGE Trailblazer view with analysis set to completed
image

**Comment by Mropat** [comment]
@henrikstranneheim ready for review - would be great to speed it up to resume usual prod activity

File: trailblazer/cli/core.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 @click.version_option(trailblazer.__version__, prog_name=trailblazer.__title__)
 @click.pass_context
 def base(context, config, database, log_level):
-    """Trailblazer - Simplify running MIP!"""
+    """Trailblazer - Tracking ongoing analyses!"""
```
Comment (henrikstranneheim) [inline trailblazer/cli/core.py:23]:
```suggestion
    """Trailblazer - Monitor analyses"""
```

File: trailblazer/cli/core.py [hunk 2]
```diff
@@ -50,16 +50,16 @@ def init(context, reset, force):
 @base.command()
 @click.pass_context
 def scan(context):
-    """Scan a directory for analyses."""
+    """Scan ongoing analyses in SLURM."""
```
Comment (henrikstranneheim) [inline trailblazer/cli/core.py:53]:
```suggestion
    """Scan ongoing analyses in SLURM"""
```

File: trailblazer/cli/core.py [hunk 3]
```diff
@@ -50,16 +50,16 @@ def init(context, reset, force):
...
     LOG.info("All analyses updated!")
 
 
-@base.command()
+@base.command("update-analysis")
 @click.argument("analysis_id")
 @click.pass_context
 def update_analysis(context, analysis_id: int):
-    """Update a single analysis."""
+    """Update status of a single analysis."""
```
Comment (henrikstranneheim) [inline trailblazer/cli/core.py:62]:
```suggestion
    """Update status of a single analysis"""
```

File: trailblazer/cli/core.py [hunk 4]
```diff
@@ -90,6 +90,17 @@ def cancel(context, analysis_id):
...
 
 
+@base.command("set-completed")
+@click.argument("analysis_id", type=int)
+@click.pass_context
+def set_analysis_completed(context, analysis_id):
+    """Set status of an analysis to "COMPLETED" """
+    try:
+        context.obj["trailblazer"].set_analysis_completed(analysis_id=analysis_id)
+    except Exception as e:
```
Comment (henrikstranneheim) [inline trailblazer/cli/core.py:100]: What does "e" stand for. Feel a bit cryptic and unsafe to just call it "e".

File: trailblazer/store/api.py [hunk 5]
```diff
@@ -344,6 +350,15 @@ def update_ongoing_analyses(self, ssh: bool = False) -> None:
                     f"Failed to update {analysis_obj.family} - {analysis_obj.id}: {e.__class__.__name__}"
                 )
 
+    def get_elapsed_time(self, analysis_obj: models.Analysis) -> str:
```
Comment (henrikstranneheim) [inline trailblazer/store/api.py:353]: Nice! We are in a hurry here, but it would be great in the future to write a unit test whenever creating a new def

**Review by henrikstranneheim — APPROVED** [review]
🚀 
Great work!

File: trailblazer/cli/core.py (hunk 4 above)
Comment (Mropat) [reply trailblazer/cli/core.py:100]: This has always been a standard thing to call error object. Ive been copying how its always been done in cg... maybe time to rethink

File: trailblazer/store/api.py (hunk 5 above)
Comment (Mropat) [reply trailblazer/store/api.py:353]: This one used to be in the comment - but as Emma asked to remove it, it is moved to inactive code block. In the future we will have a new field to keep this value

**Comment by Mropat** [comment]
deployed on stage
image

deployed on prod
image

**Pull request merged**